Snakefile

```diff
@@ -420,7 +420,7 @@ rule build_demand_profiles:
     input:
         base_network="networks/" + RDIR + "base.nc",
         regions="resources/" + RDIR + "bus_regions/regions_onshore.geojson",
-        load=load_data_paths,
+        load=get_load_paths_gegis("data", config, check_existence=True),
```

Comment:
Good :D
Then the load_data_paths object may be deleted (on top of the snakefile)